fastai/vision/data.py

```diff
@@ -20,6 +20,12 @@ def get_grid(n, nrows=None, ncols=None, add_vert=0, figsize=None, double=False,
     if title is not None: fig.suptitle(title, weight='bold', size=14)
     return (fig,axs) if return_fig else axs
 
+# Cell
+def clamp_points(samples):
+    "Clamps points to an image border"
+    def _f(imgs, pnts): return imgs,torch.clamp(pnts, -1, 1)
+    return [_f(*s) for s in samples]
```

Comment:
I don't see the benefit of defining `_f`? Why not just put that inline into the list comprehension? Or alternatively keep `_f`, and use `samples.starmap(_f)`

Reply:
I was mostly looking at the bounding box code for that one, so I can look into modifying it. I can also then potentially do the same for the bbox code too.

Reply:
I don't believe they come in as an `L` though, so that could throw an issue? Doing the following:

```python
def print_types(s): print(type(s))
dls= dblock.dataloaders(path/'images', bs=64, before_batch=print_types)
```
Shows that it winds up coming in as a `list`, unless you are suggesting something like:
```python
return L(samples).starmap(_f)
```